tests/_fixtures.py

```diff
@@ -97,6 +100,34 @@ def style_image(image_small_2):
     return image_small_2
 
 
+@pytest.fixture
+def guides(target_image):
```

Comment:
I think we should create a function `create_guides` in `_utils` that does this and use that in a fixture called `input_guides` and `target_guides`. With this we are flexible to make guides for other sized images as well.